src/wx/faudio.cpp

```diff
@@ -359,38 +355,36 @@ bool FAudio_Output::init(long sampleRate)
     soundBufferLen = (freq / 60) * 4;
     // create own buffers to store sound data because it must not be
     // manipulated while the voice plays from it
-    buffers = (BYTE*)malloc((bufferCount + 1) * soundBufferLen);
+    buffers = (uint8_t*)malloc((bufferCount + 1) * soundBufferLen);
```

Comment:
Would it make sense to switch `buffers` to be a `std::vector<yunt8_t>` instead?
You can use `resize(bufferCount + 1)` to resize it here, which will also zero out the memory at the same time. You can access the underlying C-array if you need to via the `data()` API.

Reply:
I'll follow up separately on these.